helios.js

```diff
@@ -28652,6 +28724,48 @@ class InlineDatum extends Datum {
 	}
 }
 
+class TxMetadata {
+	/**@type {Object.<number, Object.<string, any> | string>} */
```

Comment:
I would make the inner type a jsdoc typedef. So above the TxMetadata class definition you can put: `/** @typedef {Object.<string, any> | string} Metadata */`. Then you can use `Metadata` inside jsdoc comments as if it were a regular type (eg. `Object.<number, Metadata>`)